hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallSuite.java

```diff
@@ -731,6 +731,8 @@ final HapiSpec cannotUseMirrorAddressOfAliasedContractInPrecompileMethod() {
                 .then();
     }
 
+    // refusingEthConversion because the minters contract has placeholders that the HapiEthereumContractCreate doesn't
+    // support
```

Comment:
The `Minters` contract has a placeholder(`__AddressBook.sol:AddressBook___________`) that we are replacing in `bookInterpolated` before uploading the contract. When we do `HapiContractCreate` we don't use the bytecode, instead, we just pass the fileId(and later on we get the bytecode itself). For the `EthereumContractCreate` we do need the bytecode so we can construct `EthTxData`. In order to do that we have two options, as far as I can see:
1. Refactor the `EthereumContractCreate` and `EthTxData` to use the fileId
2. Give `EthereumContractCreate` access to the `HapiSpecOperations` and get the `HapiFileUpdate` by name that is replacing the empty byte contents(from create) with the corrected placeholder replaced bytecode(the logic in `createLargeFile`).
Both of these seem like an overkill to support only for a single test that we have.
There might be some more elegant way that I do not see. If you have an idea I can try it :)